maps/tether/tether_defines.dm

```diff
@@ -45,7 +45,7 @@
 
 	lobby_icon = 'icons/misc/title_vr.dmi'
 	lobby_screens = list("tether")
-	id_hud_icons = 'icons/mob/hud_jobs_vr.dmi'
```

Comment:
comment isnt really necessary

Reply:
Is it wrong to tell the next guy that's going to change the code that that line is meaningless?